### ISSUE TITLE
missing logs for prometheus-stack

### ISSUE THREAD

**Issue by schulze-b1** [osism/issues#540]
there is a folder /var/log/kolla/prometheus on the server running prometheus and the alertmanager, but the folder is empty. i already added --log.level=debug to the variable "prometheus_cmdline_extras", but it didn't help. according to the internet prometheus only logs to stderr and not to a file, which is not helpful here. and for alertmanager there doesn't seem to be a variable to add commandline extras as in log.level and i don't know if it also only logs to stderr or to a file if the option is set.

we are having issues we currently can't debug without having any logs. how can we get logs here?

**Comment by berendt** [comment]
Are there any logs when using ``docker logs prometheus_server``?

**Comment by schulze-b1** [comment]
yes, there are few logs, e.g. when the container/prometheus restarts. but even with debug enabled i only have maybe one log message a minute. i would expect way more. i do not see anything like when there is an alert in pending/firing/resolved, some metic is scraped, etc.
for the alertmanager i only see the logs for the alertmanager-restart when the container is restarted. nothing else.

**Comment by berendt** [comment]
This should be fixed today.
